samples/bluetooth/broadcast_audio_sink/Kconfig

```diff
@@ -43,4 +43,28 @@ config ENABLE_LC3
 	select LIBLC3
 	select FPU
 
+config USE_USB_AUDIO_OUTPUT
+	bool "Use USB Audio as output"
+	# By default, use the USB Audio path is disabled.
+	default n
+	depends on ENABLE_LC3
+	select USB_DEVICE_STACK
+	select USB_DEVICE_AUDIO
+	select RING_BUFFER
+
+config USE_AUTOMATIC_BROADCAST_SYNC
+	bool "Automatic Broadcast Sync"
+	default y
+	help
+	  Instead of using a broadcast assistant, sync automatically to first broadcast which
+	  has the configured TARGET_BROADCAST_CHANNEL.
+
+config TARGET_BROADCAST_CHANNEL
+	int "Broadcast Channel Audio Location to sync to"
+	# These corresponds to the bt_audio_location, supporting mono, left and right channels
+	range 0 2
+	default 1
+	help
+	  Channel Audio Location to sync to.
```

Comment:
This should depend on `USE_USB_AUDIO_OUTPUT` - This makes the sample only able to sync to a single channel, but that's only an issue for the USB stuff, right?